_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+
+## Project solutions
+
+&nbsp;&nbsp;&nbsp;&nbsp;In the first step, we started with an analysis of contents and fields that are required or suggested by the organized editing guidelines, then we check whether these are already available in the Tasking Manager, or not.
+&nbsp;&nbsp;&nbsp;&nbsp;Then we implemented a small proof of concept for testing out how the communication between [Tasking Manager](https://tasks.hotosm.org/) and OSM Community should occur. Although the OSM’s organized editing guidelines suggest the project wiki for reporting of organized mapping activities, we wanted to include two technical ideas for this initial test:
+- Report data directly to the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page)
+- Store files in a human-readable format in a git repository (GitHub, Gitlab, etc)
+
+## Validating ideas
+&nbsp;&nbsp;&nbsp;&nbsp;We participated in the [State of the Map 2020](https://2020.stateofthemap.org/) conference as a self-organized session in order to validate the project solutions initially thought and receive more feedback from the community.
+&nbsp;&nbsp;&nbsp;&nbsp;We received some suggestions on the initial ideas, mainly for reporting data to the wiki directly instead of reporting to a git repository. After discussing internally, we reached a consensus that it would be better to use both solutions, instead of just one as thought at the beginning of the project. Comming to this architecture and also, a cool name to the project - [OEG Reporter](https://github.com/hotosm/oeg-reporter). <img src="https://user-images.githubusercontent.com/18130942/91627404-7ae96c80-e98d-11ea-98a7-999c754cd19f.png" />
+
+
+## Implementing
+&nbsp;&nbsp;&nbsp;&nbsp;After defining with the community the solutions to be used, we implemented the data report for a git repository and an instance of MediaWiki (the system used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page)) in a flask API, mainly using libraries to format data according to the format used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page) (wikitextparser) and upload and update files in a git repository (gitpython).
+&nbsp;&nbsp;&nbsp;&nbsp;The most challenging part of the project's development was definitely to implement data reporting for the wiki, due to the file format used by the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page) having a slightly more complex syntax, but after a few weeks of study this was possible. [OEG reporter](https://github.com/hotosm/oeg-reporter) integration with Tasking Manager can be seen in the next images.<br>
+* User select if the organisation is going to enable the automated OEG Report
```

Comment:
I prefer to use `###` instead of `*`. Add blank lines before and after title and img

Reply:
I updated all the text to use sections instead of lists